src/commons/application/reducers/__tests__/SessionReducer.ts

```diff
@@ -546,11 +559,17 @@ const gradingOverviewTest2: GradingOverview[] = [
 test('UPDATE_GRADING_OVERVIEWS works correctly in inserting grading overviews', () => {
   const action = {
     type: UPDATE_GRADING_OVERVIEWS,
-    payload: gradingOverviewTest1
-  };
+    payload: {
+      count: gradingOverviewTest1.length,
+      data: gradingOverviewTest1
+    }
+  } as const;
```

Comment:
I will definitely have to adjust the tests here as I tweak the frontend fields within GradingOverviews - I will add my changes if this merges after our pull request! If merged before the pull request, I will adjust the test cases within my pull.